app/javascript/onboarding/Onboarding.jsx

```diff
@@ -47,6 +45,8 @@ export default class Onboarding extends Component {
       this.setState({
         currentSlide: nextSlide,
       });
+    } else {
+      window.location.href = '/';
```

Comment:
Might just want to include a comment here since it feels a bit random to change the window's location to `/` without any context

Reply:
We can refine this in https://github.com/thepracticaldev/dev.to/issues/6544

Reply:
you can also use window.location.assign("/") which is a little bit more explicit

Reply:
I'm going to be in the onboarding code today and will be making a PR, so I can swap `href` out for `assign`!